drivers/aip31068/include/aip31068_regs.h

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (C) 2020 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_aip31068
+ * @brief       Register definitions for the AIP31068 I2C LCD controller
+ * @author      Hendrik van Essen <hendrik.ve@fu-berlin.de>
+ * @file
+ * @{
+ */
+
+#ifndef AIP31068_REGS_H
+#define AIP31068_REGS_H
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+/* Commands (page 18, table 3) */
+
+/**
+ * @brief Clears entire display and sets cursor to position (0, 0).
+ *
+ * ATTENTION: Also changes BIT_ENTRY_MODE_INCREMENT to 1
+ */
+#define CMD_CLEAR_DISPLAY   0x01
+
+/**
+ * @brief Sets cursor to position (0, 0) and resets display to original position
+ *        before any shift operations.
+ */
+#define CMD_RETURN_HOME 0x02
+
+/**
+ * @brief Sets cursor move direction and specifies display shift.
+ */
+#define CMD_ENTRY_MODE_SET  0x04
+
+/**
+ * @brief Sets entire display on/off, cursor on/off, and blinking of cursor
+ *        position character on/off.
+ */
+#define CMD_DISPLAY_CONTROL 0x08
+
+/**
+ * @brief Moves cursor and shifts display.
+ */
+#define CMD_CURSOR_DISPLAY_SHIFT    0x10
+
+/**
+ * @brief Sets interface data length, number of display lines and character
+ *        font size.
+ */
+#define CMD_FUNCTION_SET    0x20
+
+/**
+ * @brief Sets CGRAM address.
+ */
+#define CMD_SET_CGRAM_ADDR  0x40
+
+/**
+ * @brief Sets DDRAM address.
+ */
+#define CMD_SET_DDRAM_ADDR  0x80
+
+
+
+/* Bits for CMD_ENTRY_MODE_SET (page 16, section 3) */
+
+/**
+ * @brief 0 = Decrement cursor after insertion
+ *        1 = Increment cursor after insertion
+ */
+#define BIT_ENTRY_MODE_INCREMENT    1
+
+/**
+ * @brief 0 = No automated display scroll
+ *        1 = Automated display scroll
+ */
+#define BIT_ENTRY_MODE_AUTOINCREMENT    0
+
+
+
```

Comment:
Are the extra spaces on purpose?

Reply:
Yes it's on purpose, personally I like to have somewhat of a visual break between different groups of values.

Reply:
Ok, your call.